crates/rpc/rpc/Cargo.toml

```diff
@@ -42,6 +42,7 @@ jsonrpsee.workspace = true
 http = "0.2.8"
 http-body = "0.4.5"
 hyper = "0.14.24"
+reqwest = "0.11.20"
```

Comment:
this should be optional

Reply:
Addressed in https://github.com/anton-rs/op-reth/pull/151